webpack.config.js

```diff
@@ -89,7 +89,7 @@ module.exports = {
   performance: {
     hints: false
   },
-  devtool: '#eval-source-map'
+  devtool: 'cheap-module-source-map'
```

Comment:
this is the only dev source-map option i could find that provided actual human readable vue files in the dev deployment for debugging...